pre-flight-checks.sh

```diff
@@ -8,8 +8,8 @@ ERRORS=()
 
 echo " *** Checking that site can be built."
 
-wok --serve >/dev/null 2>&1 &
-WOK_PID=$!
+python manage.py serve >/dev/null 2>&1 &
```

Comment:
Now that we've got Django under the hood (pun, so very much intended ;) ) - we could also use the checks framework as an initial test here.